core/js/js.js

```diff
@@ -1135,6 +1135,7 @@ OC.Notification = {
 	 * @param {string} html Message to display
 	 * @param {Object} [options] options
 	 * @param {string] [options.type] notification type
+	 * @param {string] [options.hideCloseButton] hide close button
```

Comment:
```suggestion
	 * @param {string} [options.type] notification type
	 * @param {string} [options.hideCloseButton] hide close button
```
Bracket-pairing needs fixing.
(and also comment above by @DeepDiver1975